docs/source/release_notes.rst

```diff
@@ -7,6 +7,7 @@ Future Release
 ==============
     * Enhancements
         * Add ``TimeSinceLastFalse``, ``TimeSinceLastMax``, ``TimeSinceLastMin``, and ``TimeSinceLastTrue`` primitives (:pr:`2418`)
+        * Add MaxConsecutiveFalse, MaxConsecutiveNegatives, MaxConsecutivePositives, MaxConsecutiveTrue, MaxConsecutiveZeros, NumConsecutiveGreaterMean, NumConsecutiveLessMean (:pr:`2420`)
```

Comment:
nit: Include class names in double back-tics for consistent formatting.

Reply:
Fixed